robonomics/io/src/sink.rs

```diff
@@ -19,3 +19,4 @@
 
 // Collection of virtual actuators (like stdout).
 pub mod virt;
+pub mod ipfs;
```

Comment:
I guess is better to include IPFS into `virt` module, because IPFS is a virtual I/O device.